definitions/divorce/json/ComplexTypes/ComplexTypes-nonprod.json

```diff
@@ -1,10 +1 @@
-[
```

Comment:
this non-prod file, should be deemed-and-dispensed-nonprod since it was added in https://tools.hmcts.net/jira/browse/DIV-6627 and this ticket belong to D&D epic.